sdk/lib/atl/atlbase.h

```diff
@@ -662,6 +665,127 @@ class CAtlDllModuleT : public CAtlModuleT<T>
     }
 };
 
+
+template <class T>
+class CAtlExeModuleT : public CAtlModuleT<T>
+{
+public:
+    DWORD m_dwMainThreadID;
+    //DWORD m_dwTimeOut;
+    //DWORD m_dwPause;
+    //bool m_bDelayShutdown;
+
+    CAtlExeModuleT()
+    {
+        m_dwMainThreadID = ::GetCurrentThreadId();
```

Comment:
A member initializer list should be preferred over assignments.

Reply:
Yeah in this case it makes sense.